frontend/integration-tests/tests/dashboards/project-dashboard.scenario.ts

```diff
@@ -35,7 +35,7 @@ const launcherLink = {
     namespaceDashboard: {
       namespaces: [testName],
     },
-    text: 'Help Menu Link',
+    text: 'Namespace Dashboard Link',
```

Comment:
This change looks unrelated

Reply:
Sorry, screwed up with `--amend` while trying to squash local commits. Starting over..

Reply:
Okay, commits squashed.